guides/doc-Administering_Red_Hat_Satellite/topics/Renaming_a_Satellite_Server.adoc

```diff
@@ -8,6 +8,13 @@ If you use external authentication, you must reconfigure {ProjectServer} for ext
 The `{project-change-hostname}` script breaks external authentication for {ProjectServer}.
 For more information about configuring external authentication, see xref:chap-{Project_Link}-Administering_{Project_Link}-Configuring_External_Authentication[].
 
+[IMPORTANT]
```

Comment:
This has not been standard practice throughout Satellite docs.

Reply:
Why note in one file and important in another? 
I'm trying to follow the logic here.
Everything mentioned in the docs is important and noteworthy; labelling something as important on a page full of equally important information seems like a poor implementation of the principles of minimalism. 
When I read the text above and below the admonition in the context of the rest of the information that precedes the prerequisites, I've no idea why this is called out. 
If an engineer says to add a note, it doesn't necessarily mean that a writer has to add a note with a note label.
I hope you get what I'm trying to say. 

I mention this because with a project as old and complex as ours - Foreman being 12 years old this year -  there are a lot of points to consider, a lot of "notes" to be added. To ensure the quality of the docs, we should ensure that we maintain order. What I'm seeing here is a brief introduction, then random admonitions, then another admonition highlighted, yet not greater in importance than those that come before and after. 
Hope this makes sense.